.github/PULL_REQUEST_TEMPLATE/release.md

```diff
@@ -0,0 +1,5 @@
+# ELements Release PR Template
+
```

Comment:
How does that release template work? Does it replace content when there is a new release?